pkg/i2gw/providers/kong/kong.go

```diff
@@ -34,15 +34,18 @@ func init() {
 
 // Provider implements the i2gw.Provider interface.
 type Provider struct {
-	storage        *storage
-	resourceReader *resourceReader
-	converter      *converter
+	conf    *i2gw.ProviderConf
+	storage storage
+
+	*resourceReader
+	*converter
 }
 
 // NewProvider constructs and returns the kong implementation of i2gw.Provider.
 func NewProvider(conf *i2gw.ProviderConf) i2gw.Provider {
 	return &Provider{
-		storage:        newResourcesStorage(),
+		conf: conf,
```

Comment:
I dont think we ever use this. 
the resourceReader is the one that holds the conf

Reply:
yep, removed